app/src/main/java/husaynhakeem/io/facedetectorapp/MainActivity.kt

```diff
@@ -42,24 +59,27 @@ class MainActivity : AppCompatActivity() {
 
     private fun setupCamera(lensFacing: Facing) {
         val faceDetector = FaceDetector(faceBoundsOverlay)
+        faceDetector.setonFaceDetectionFailureListener(onFaceDetectionResultListener)
         viewfinder.facing = lensFacing
+        viewfinder.rotation = 180f
         viewfinder.addFrameProcessor {
             faceDetector.process(
                 Frame(
                     data = it.data,
                     rotation = it.rotation,
                     size = Size(it.size.width, it.size.height),
                     format = it.format,
-                    lensFacing = if (viewfinder.facing == Facing.BACK) LensFacing.BACK else LensFacing.FRONT
+                    lensFacing = LensFacing.FRONT
```

Comment:
Could you set the value of the frame's lensFacing parameter to the variable `lensFacing`?